src/main/java/com/github/manolo8/darkbot/gui/utils/UIUtils.java

```diff
@@ -7,6 +7,7 @@
 import javax.swing.border.Border;
 import javax.swing.border.MatteBorder;
 import java.awt.*;
+import java.util.Objects;
 
 public class UIUtils {
```

Comment:
are any of the changes in UIUtils really needed? can't see them used anywhere

Reply:
think it was auto import for something, cause of ide

Reply:
wait its in 

`public static ImageIcon getIcon(String name) {
        return new ImageIcon(new ImageIcon(Objects.requireNonNull(UIUtils.class.getResource("/" + name + ".png"))).getImage()
                .getScaledInstance(16, 16, Image.SCALE_SMOOTH));
    }`

Reply:
i mean the changes in the class as a whole, not the import